lyluatex.sty

```diff
@@ -30,9 +30,12 @@
     ['fragment'] = {'true', 'false', ''},
     ['fullpagealign'] = {'staffline', 'crop'},
     ['fullpagestyle'] = {''},
+    ['gutter'] = {'.4in', ly.is_dim},
+        ['exampleindent'] = {'gutter', ly.is_alias},
+        ['leftgutter'] = {'', ly.is_dim}, ['rightgutter'] = {'', ly.is_dim},
```

Comment:
I will have to reconsider the use of the gutters when I (finally) get to rethinking about the whold protrusion thing, but for now I'd say this is good to go.